cellxgene_schema_cli/cellxgene_schema/ontology_files/owl_info.yml

```diff
@@ -1,23 +1,23 @@
 CL:
-  latest: 2023-06-22
+  latest: 2023-08-24
   urls:
-    2023-06-22: https://github.com/obophenotype/cell-ontology/releases/download/v2023-06-22/cl.owl
+    2023-08-24: https://github.com/obophenotype/cell-ontology/releases/download/v2023-08-24/cl.owl
 EFO:
-  latest: 2023-07-17 EFO 3.56.0
+  latest: 2023-08-15 EFO 3.57.0
   urls:
-    2023-07-17 EFO 3.56.0: https://github.com/EBISPOT/efo/releases/download/v3.56.0/efo.owl
+    2023-08-15 EFO 3.57.0: https://github.com/EBISPOT/efo/releases/download/v3.57.0/efo.owl
 HANCESTRO:
-  latest: 2022-07-18 (2.6)
+  latest: 3.0
   urls:
-    2022-07-18 (2.6): https://github.com/EBISPOT/ancestro/raw/2.6/hancestro.owl
+    3.0: https://github.com/EBISPOT/hancestro/blob/3.0/hancestro-base.owl
```

Comment:
URL should be a download link to the raw file rather than github (https://github.com/EBISPOT/hancestro/raw/3.0/hancestro-base.owl)

Reply:
great catch. my copy operation failed